Lib/test/test_uuid.py

```diff
@@ -674,26 +674,57 @@ class TestUUIDWithExtModule(BaseTestUUID, unittest.TestCase):
 class BaseTestInternals:
     _uuid = py_uuid
 
+    # data is specific to AIX - with '.' as _MAC_DELIM
+    # and strings shorter than 17 bytes (no leading 0)
+    @unittest.skipUnless(_AIX, 'requires AIX')
+    # key is on lineX, value is on lineX+1 aka 'nextline'
+    def test_find_mac_nextlines(self):
+        data = '''\
+Name  Mtu   Network     Address           Ipkts Ierrs    Opkts Oerrs  Coll
+en0   1500  link#2      fe.ad.c.1.23.4   1714807956     0 711348489     0     0
+                        01:00:5e:00:00:01
+en0   1500  192.168.129 x071             1714807956     0 711348489     0     0
+                        224.0.0.1
+en0   1500  192.168.90  x071             1714807956     0 711348489     0     0
+                        224.0.0.1
+'''
+        popen = mock.MagicMock()
+        popen.stdout = io.BytesIO(data.encode())
+
+        with mock.patch.object(shutil, 'which',
+                                        return_value='/usr/bin/netstat'):
+            with mock.patch.object(subprocess, 'Popen',
+                                            return_value=popen):
+                mac = self.uuid._find_mac_nextlines(
+                    command='netstat',
+                    args='-ia',
+                    hw_identifiers=b'Address',
+                    f_index=lambda x: x,
+                )
+
+        self.assertEqual(mac, 0xfead0c012304)
+
     @unittest.skipUnless(os.name == 'posix', 'requires Posix')
-    def test_find_mac(self):
+    @unittest.skipIf(_AIX, 'AIX has different DELIM')
```

Comment:
Rough inverse of the above suggestion:

    @unittest.skipUnless(self.uuid._MAC_DELIM == b':')